<launch>

    <rosparam file="$(find ee_cart_imped_control)/pr2_arms_cart_imped_controller.yaml" command="load" />
    
    <!-- Left arm -->
    <node pkg="pr2_controller_manager" type="spawner" args="l_arm_cart_imped_controller" name="l_arm_cart_imped_controller_spawner" output="screen"/>
    <group ns="l_arm_cart_imped_controller">
	       <node pkg="ee_cart_imped_action" name="l_arm_cart_imped_action_node" type="ee_cart_imped_action" output="screen"/>
	</group>
    <!-- Right arm -->
    <rosparam file="$(find ee_cart_imped_control)/pr2_arms_cart_imped_controller.yaml" command="load" />
    <node pkg="pr2_controller_manager" type="spawner" args="r_arm_cart_imped_controller" name="r_arm_cart_imped_controller_spawner" output="screen"/>
    <group ns="r_arm_cart_imped_controller">
	       <node pkg="ee_cart_imped_action" name="r_arm_cart_imped_action_node" type="ee_cart_imped_action" output="screen"/>
	</group>


</launch>
